<p><strong>Full-length transcript sequencing of human and mouse identifies widespread isoform diversity and alternative splicing in the cortex</strong></p>
<p>The full text on Cell Reports is <a href="https://www.cell.com/cell-reports/fulltext/S2211-1247(21)01504-7">here</a>.</p>
<p>A Genome Browser Track Hub can be found <a href="http://genome.ucsc.edu/cgi-bin/hgTracks?hubUrl=http://genome.exeter.ac.uk/hub/hub.txt">here</a>. Tracks available on human (hg38) and mouse (mm10) builds.</p>
<p><img src="BrainIsoforms_files/image001.png" alt="" />&nbsp;</p>
<p>An isoform viewer resource (beta) can be found <a href="build/index.html">here</a></p>
<p>&nbsp;<img src="BrainIsoforms_files/image003.jpg" alt="" /><br /> </p>
<p>Hub Notes:</p>
<p>The online resource is provided as a UCSC Genome Browser Track Hub displaying both SQANTI2 filtered and SQANTI2 unfiltered isoform structure data. Additional annotation through the use of coloring to indicate structural categories as described below (original source <a href="https://github.com/Magdoll/SQANTI2">https://github.com/Magdoll/SQANTI2</a> )</p>
<ul>
<li>FSM (<em>Full Splice Match</em>): meaning the reference and query isoform have the same number of exons and each internal junction agree. The exact 5' start and 3' end can differ by any amount.</li>
<li>ISM (<em>Incomplete Splice Match</em>): the query isoform has fewer 5' exons than the reference, but each internal junction agree. The exact 5' start and 3' end can differ by any amount.</li>
<li>NIC (<em>Novel In Catalog</em>): the query isoform does not have a FSM or ISM match, but is using a combination of known donor/acceptor sites.</li>
<li>NNC (<em>Novel Not in Catalog</em>): the query isoform does not have a FSM or ISM match, and has at least one donor or acceptor site that is not annotated.</li>
<li><em>Antisense</em>: the query isoform does not have overlap a same-strand reference gene but is anti-sense to an annotated gene.</li>
<li><em>Genic Intron</em>: the query isoform is completely contained within an annotated intron.</li>
<li><em>Genic Genomic</em>: the query isoform overlaps with introns and exons.</li>
<li><em>Intergenic</em>: the query isoform is in the intergenic region.</li>
</ul>
<p>If a transcript is associated with a known gene, the associated gene name will be show in place of the original PB.1234.4 original annotation from Iso-Seq. The number of reads suupporting that isoform are also shown. For example:&nbsp;&nbsp; <u>NNAT.3/6</u>&nbsp; indicates an isoform associated with the gene NNAT. It is isoform number 3 and shows 6 independent full length reads supporting this isoform structure. <br /> </p>
<p>Related to number of reads supporting each transcript isoform, we have also created alternate 'Read' tracks to indicate levels of expression through shading, where a light shaded isoform indicates a minimal support of just 2 reads, with intensity increasing as more and more reads support each isoform.<br /> </p>
<p>RNA-Seq data when available is also shown as RPKM coverage based on a 10bp window. <br /> </p>